README.md

```diff
@@ -79,6 +79,27 @@ yarn run lint
 
 Test all the packages in the monorepo.
 
+Some tests will require services to be running locally
+
+```bash
+docker compose up
+```
+
+Deploy Smart Contracts
+
+```bash
+yarn run deploy:contracts
+```
```

Comment:
This step could be automated by adding the following to the docker-compose:

```
  request-contracts:
    image: requestnetwork/request-contracts
    container_name: request-contracts
    environment:
      WEB3_URL: http://ganache:8545
    depends_on:
      - ganache
```

; but the drawback is that it won't use the smart contract code checked out in the repository, as it will be using the one from the Docker image.

Reply:
Yes, I started using it but I decided to have test coverage for the current smart contract source code